test/test_CMSSM_unitarity.m

```diff
@@ -37,5 +37,5 @@
 
 unitarity = FSCMSSMCalculateUnitarity[handle];
 
-TestCloseRel[FlexibleSUSYUnitarity`RenormalizationScale /. unitarity[[1,2]], 866.8013526495104, 10^-16];
-TestCloseRel[FlexibleSUSYUnitarity`MaxAbsReEigen /. unitarity[[1,2]], 0.02973890765470515, 3.*10^-16];
+TestCloseRel[FlexibleSUSYUnitarity`RenormalizationScale /. unitarity[[1,2]], 866.786, 10^-6];
```

Comment:
Can I ask why did you reduce this precision?

Reply:
Yes, I also stumbled upon this. I think the deviation that we see popping up in this test lies in the following: 
* At the low-energy scale we calculate $\alpha_{em}$ and $\theta_w(G_F,M_W)$. From these two parameters we calculate $g_1, g_2$.
* Now, the value of $\theta_w(G_F,M_W)$ depends on the W boson pole mass $M_W$. Before this PR we took this value for $M_W$ from `qedqcd`, which stores some default or user-given input value.
* However, what I think we actually have to do is to take the predicted value of $M_W$ and feed it as input into the calculation of $\theta_w(G_F,M_W)$. This is what this PR proposes.

The effect of this PR is, that the value of $\theta_w$ changes slightly (but not much). This leads to a small change in $g_1$ and $g_2$ and thus to a slight change in all dependent quantities, such as pole masses, observables, ... So, in particular this test now fails.

Since the effect is basically understood, I thought it would be ok to simply update the tested values.

In addition it seems to me that the original test precision of $10^{-16}$ is much smaller than the precision goal of the spectrum generator output (usually ~$10^{-4}$--$10^{-5}$). Thus, I think it makes in general more sense to test against fixed values with a precision that is around the spectrum generator's precision goal. (Of course there could be well-motivated exceptions to this rule of thumb, I'd say.)

Are you unhappy with this reduction of precision?

Reply:
I would think its hard fro the renormlaisation scale to be more precise if its set by the geometric mean of the stop masses as these should be sensitive to the iteration precision and also to fundamental limitation son how precise you can get the Runga-Kutta to be.  But then I am a bit confused as to why it was working with 10^{-10} precision before?

Reply:
@pathron Good point. I think this is basically also what I was arguing, namely that the test precision should be of the order of the overall precision goal.

I think it was working before with 10^{-10} because as long as no details of the calculation are changed, the results are always identical, so one could in principle test with a very high precision. Then one would be sensitive to any subtle change in the code where the tests would fail. (This latter approach could in principle be used to detect such subtle changes in the calculations.)